docs/source/host/server-setup/linux-server.md

```diff
@@ -31,7 +31,7 @@ For the list of network ports that you will need to open and manage, see [here](
 
 Connect a web browser to the server at port 40100. (If you are on the machine that the server is running on, this would be http://localhost:40100) Complete the initial setup wizard and you should have a functioning domain.
 
-## Files
+## Files and Package Configuration
```

Comment:
```suggestion
## Files and Server Configuration
```
While it is how we configured the package, the user will be interested in where the server configuration is and not how we configured the package.